tools/check_format_test_helper.py

```diff
@@ -169,6 +169,7 @@ def checkFileExpectingOK(filename):
   errors += checkFileExpectingError("proto_format.proto", "clang-format check failed")
   errors += checkFileExpectingError("real_time_source.cc", real_time_inject_error)
   errors += checkFileExpectingError("real_time_system.cc", real_time_inject_error)
+  errors += checkFileExpectingError("std_atomic_free_functions.cc", "std::atomic_*")
```

Comment:
can you also add a check for fixFileExpectingFailure? This error is not automatically fixable.